packages/next/src/lib/mkcert.ts

```diff
@@ -69,8 +69,27 @@ export async function createSelfSignedCertificate(
   host?: string,
   certDir: string = 'certificates'
 ): Promise<SelfSignedCertificate | undefined> {
+  // Executable name
+  const executableName = 'mkcert'
+  let binaryPath = ''
+  
   try {
-    const binaryPath = await downloadBinary()
+    // Try to find the executable path using 'which' command
+    binaryPath = execSync(`which ${executableName}`, { encoding: 'utf-8' }).trim()
+
+    // Check if the executable file exists and is executable
+    fs.accessSync(executablePath, fs.constants.X_OK)
+
+    binaryPath = executableName
+    Log.info(
+      `If the globally executable "${executableName}" is found, just use it.`
+    )
```

Comment:
```suggestion
```
I think we can omit this log as it isn't info the user needs to take action on/know per-se